build/yamls/antrea-windows-containerd.yml

```diff
@@ -5,9 +5,18 @@ data:
     mkdir -force c:/var/log/antrea
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
-    mkdir -force C:/var/run/secrets/kubernetes.io/serviceaccount
-    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/ca.crt C:/var/run/secrets/kubernetes.io/serviceaccount
-    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/token C:/var/run/secrets/kubernetes.io/serviceaccount
+    $localPath="C:/var/run/secrets/kubernetes.io/serviceaccount"
```

Comment:
Prefer meaningful name like $serviceAccountPath, since from 1.7 containerd cri c:\var\run\secrets\kubernetes.io\serviceaccount will get automatically created, add a comment mentioning that in containerd 1.6, the path does not exist and needs to be created